lib/src/mountutil.rs

```diff
@@ -7,12 +7,18 @@ use anyhow::Result;
 use cap_std::fs::Dir;
 use cap_std_ext::cap_std;
 
+// Fix musl support
```

Comment:
I'm fine with this as is but probably the best place to fix this is adding the definitions into rustix